### ISSUE TITLE
At the end of the reading page (error "there are no earlier entries" or ?skip=980), switch paid accounts to ?date= mode 

### ISSUE THREAD

**Issue by azurelunatic** [dreamwidth/dw-free#1569]
Dreamwidth displays entries from the last 2 weeks on the reading page (or ?skip=980, or whatever skip number is appropriate based on the reading page size), whichever comes first), figure out and display the appropriate date to keep reading. There may have to be some overlap. 

This is a paid feature. 

The ?date= format is in YYYY-MM-DD

**Comment by kareila** [comment]
It involves turning on the "friendspage_per_day" usercap for paid accounts.  *cringe*

**Comment by azurelunatic** [comment]
The ability to do it is already live on the site, it's just not discoverable or well-documented.

**Comment by kareila** [comment]
Oh weird, I thought if the user had that cap, it would change the "no earlier entries" text to a date skip link.

**Comment by azurelunatic** [comment]
I'm sure there was something in Bugzilla about it.

**Comment by kaberett** [comment]
Via a suggestion you made, apparently! With various headers snipped...

         bug_id: 2370
        subject: Offer view-reading-page-by-date links when there are no more ?skip= to show
               URL: http://dw-suggestions.dreamwidth.org/249418.html
          Keywords: effort-minor, from-suggestions, paiduser-enhancement,
                    why-usability

The reading page is available by date to paid users, by adding ?date=YYYY-MM-DD
instead of ?skip=N. Why not link to the appropriate date after the page has run
out of entries? 

Description:

Entries no longer show in the normal flow of the reading page after 2
weeks/1000 entries, whichever comes first. However, paid users can view their
reading pages by date (though at least in my style there's no link to other
days, just other pages of entries on that day). 

This feature is relatively little-known, and one does have to remember the date
form. Offering the links to users who can use them would be a perfectly natural
and intuitive way for people who have never heard of it to discover it. 

(Offering the links and then displaying a message that you have to have a paid
account to use them might be mean; displaying a message about the feature in
lieu of a tantalizing link might be good incentive for an upgrade.) 

In order to avoid skipping entries, the link should be to the same day as the
last entry on the page, even though this would cause some overlap. This might
cause a bit of confusion, but a bit of confusion is better than skipped
entries. (Though if it is easy to tell which entries would overlap and offer
the date link to skip that many, that would be really shiny.) 

Suggested by:

azurelunatic

**Comment by kareila** [comment]
Yeah, I was thinking of the behavior of the recent entries page past 1000 items.  When I saw read-by-date support in the reading page code, I thought it would behave the same way.

**Comment by azurelunatic** [comment]
Several news-comment requests for the existing feature from new Tumblr users.
